src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -74,16 +73,16 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
 
       // Let's slide out a root branch now
       project.contextMenu.openContextMenu("develop")
-      project.contextMenu.slideOut()
+      project.contextMenu.slideOutDefensive("develop")
       project.acceptBranchDeletionOnSlideOut()
       branchAndCommitRowsCount = project.refreshModelAndGetRowCount()
       // 5 branch rows (`develop` is no longer there) + 7 commit rows
       // (1 commit of `allow-ownership-link` and 3 commits of `call-ws` are all gone)
       Assert.assertEquals(13, branchAndCommitRowsCount)
 
       project.checkoutBranch("master")
-      project.contextMenu.openContextMenu("call-ws")
-      project.contextMenu.slideOut()
+      // project.contextMenu.openContextMenu("call-ws")
+      project.contextMenu.slideOutDefensive("call-ws")
```

Comment:
Now `slideOutDefensive` opens the context menu itself... coz I've noticed that the menu doesn't always open after the first attempt. But maybe this retry logic can be moved to `openContextMenu()` instead